README.md

```diff
@@ -29,11 +29,25 @@ There is also an experimental binary which uses Tessera's POSIX storage backend.
 
 TesseraCT is under active development, and will reach alpha in 2025Q3 🚀.
 
-At the moment, TesseraCT supports Amazon Web Service and Google Cloud Platform.
-Read the FAQ to understand [why we chose these platforms](#why-these-platforms),
-or if [you're interested in others](#can-i-run-tesseract-outside-gcp-or-aws).
+|Platform         |Architecture            |Our use-case          |Performance|Binary                                  |Deployment                                                      |
+|-----------------|------------------------|----------------------|-----------|----------------------------------------|----------------------------------------------------------------|
+|GCP              |Spanner + GCS + MIG     |public staging logs   |TODO       |[gcp](/cmd/tesseract/gcp/main.go)       |[doc](/deployment/live/gcp/static-ct-staging/logs/arche2025h1/) |
+|GCP              |Spanner + GCS + CloudRun|continuous integration|TODO       |[gcp](/cmd/tesseract/gcp/main.go)       |[example](/deployment/live/gcp/static-ct/logs/ci)               |
+|GCP              |Spanner + GCS + VM      |codelab, load tests   |TODO       |[gcp](/cmd/tesseract/gcp/main.go)       |[doc](/deployment/live/gcp/test/)                               |
+|AWS              |RDS + S3 + ECS          |continuous integration|TODO       |[aws](/cmd/tesseract/aws/main.go)       |[example](/deployment/live/aws/conformance/ci/)                 |
+|AWS              |RDS + S3 + VM           |codelab, load tests   |TODO       |[aws](/cmd/tesseract/aws/main.go)       |[doc](/deployment/live/aws/test/)                               |
+|POSIX            |ZFS + VM                |one-off test          |NA         |[posix](/cmd/experimental/posix/main.go)|[doc](/cmd/experimental/posix/)                                 |
+|Custom S3 + MySQL|MinIO + MySQL + VM      |one-off test          |NA         |[aws](/cmd/tesseract/aws/main.go)       |[doc](/docs/architecture/NONCLOUD.md)                           |
 
-[Public test instances](#test_tube-public-test-instances) run on GCP.
+These deployments come with different levels of maturity depending on
+our use-case.
+We dedicated most of our time to the GCP with Spanner + GCS + MIG implementation
```

Comment:
How about:

```suggestion
Our primary focus so far has been on the GCP with Spanner + GCS + MIG configuration
since we use it for our [public staging logs](#test_tube-public-test-instances). However we believe all implementations are correct, and we'd love to [hear your feedback](#wave-contact) on any of these implementations.

```